### DISCUSSION TITLE
Question about sensitive data in manticore-backup dump

### DISCUSSION THREAD

**Discussion by d47081** [manticoresoftware/manticoresearch-php#168] [Q&A] [answered]
I'm running web search instance with open database (there is no personal data, IP etc, just pages URL with meta data collected) and time-to-time share it with others using torrent.

Before, it was MySQL + Sphinx server so I did backup with `mysqldump` and shared it with others on time intervals.

So when I have changed engine to the Manticore DB only, my question is can I safely share `manticore-backup` folder or it may include some sensitive server data? There is lot of files but looks like nothing important there to make this index to be open for public? Or it may collect sensitive server meta info?

**Comment by sanikolaev** [comment, marked as answer]
Have you tried using mysqldump against Manticore? https://manual.manticoresearch.com/Securing_and_compacting_a_table/Backup_and_restore#Backup-and-restore-with-mysqldump

**Reply by d47081** [reply]
Not yet, thank you for advice!
